sensor/kubernetes/listener/resources/secrets.go

```diff
@@ -290,6 +307,11 @@ func (s *secretDispatcher) processDockerConfigEvent(secret, oldSecret *v1.Secret
 				registryAddress, secret.GetNamespace(), secret.GetName())
 		}
 
+		if err := validateSecret(registryAddr, dce, secret.GetNamespace(), secret.GetName()); err != nil {
+			log.Warnf("Skipping dockerConfig secret: %v", err)
```

Comment:
Be good log the namespace + secret name as well (so that can locate the bad data)

Reply:
The `err` contains details about the name of secret and namespace. This line just wraps that error and prints it, so the information will be there.